app/views/shared/banners/_parents_carers_pupils.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-warning-text govuk-!-margin-bottom-2">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
+      Parents, carers and pupils
+    </h2>
+    <p class="govuk-body">Parents, carers and pupils cannot apply for laptops, tablets or internet access. You should contact your school for help accessing remote education.</p>
+  </strong>
+</div>
```

Comment:
Could have used https://dfe-digital.github.io/govuk-components/#warning

Reply:
How can you set custom HTML within the warning text with that?